src/js/icons/StatusCriticalSmall.js

```diff
@@ -3,7 +3,7 @@ import React, { forwardRef } from 'react';
 import { StyledIcon } from '../StyledIcon';
 
 const StatusCriticalSmall = forwardRef((props, ref) => (
-  <StyledIcon ref={ref} viewBox="0 0 12 12" a11yTitle="StatusCriticalSmall" {...props}>
+  <StyledIcon ref={ref} viewBox="0 0 12 12" a11yTitle="StatusCritical" {...props}>
```

Comment:
Should we be less machine-like and more human and structure it like "critical status"?